ROSCO_toolbox/ROSCO_testing/run_Testing.py

```diff
@@ -0,0 +1,225 @@
+                print('{} is not in load rankings'.format(meas))
+        print('----------------------------------------')
+
+    else:
+        # Save each set of outputs in a list
+        allstats = []
+        allrankings = []
+        for i in range(max(len(discon_files), len(rosco_binaries))):
+            # Define output file folder
+            outputDir = os.path.join(os.getcwd(), rt.runDir,'controller_{}'.format(i)) # NJA - this is dependent on the hard-coded folder names in ROSCO_testing.py
+
+            # Load output file paths using case matrices
+            case_info = FileTools.load_yaml(os.path.join(outputDir, 'case_matrix.yaml'), package=1)
+            casenames = case_info['Case_Name']
+            if rt.outfile_fmt >= 2:
+                outFileNames = [os.path.join(outputDir, fname+'.outb') for fname in casenames]
+            else:
+                outFileNames = [os.path.join(outputDir, fname+'.out') for fname in casenames]
+
+            # Set some processing parameters
+            fp.OpenFAST_outfile_list = outFileNames
+            fp.t0 = 100
+            if rt.cores > 0:
+                fp.parallel_analysis = True
+            fp.results_dir = os.path.join(outputDir, 'stats')
+            fp.verbose = True
+            fp.save_LoadRanking = True
+            fp.save_SummaryStats = True
+
+            # Define load ranking stats
+            fp.ranking_vars = [['RotSpeed'],
+                            ['RootMyb1', 'RootMyb2', 'RootMyb3'],
+                            ['TwrBsMyt']]
+            fp.ranking_stats = ['max',
+                                'max',
+                                'max']
+            if turbine2test == 'IEA-15MW':
+                fp.ranking_vars += [['PtfmPitch']]
+                fp.ranking_stats += ['max']
+
+            # Load and save statistics and load rankings
+            if not os.path.exists(os.path.join(fp.results_dir, 'dataset1_LoadRanking.yaml')) or reCrunch:
+                stats, load_rankings = fp.batch_processing()
+            
+            # Save all outputs
+            allstats.append(stats)
+            allrankings.append(load_rankings[0])
+
+        # Print a summary
+        print('----------------------------------------')
+        print('\t\t{}'.format(turbine2test))
+        print('----------------------------------------')
+        print('\t\t{}\t{}'.format('Controller1', 'Controller2'))
+        for meas in allrankings[0].keys():
+            try:
+                print('Max {} \t{:1.3e}\t{:1.3e}'.format(
+                    meas, allrankings[0][meas]['max'][0], allrankings[1][meas]['max'][0]))
+            except:
+                print('{} is not in load rankings'.format(meas))
+        print('----------------------------------------')
+        
+if __name__ == "__main__":
+
+    # WEIS directory, for running openfast, etc.
+    weis_dir = os.environ.get('weis_dir')       # works if we do  `export weis_dir=$(pwd)` in WEIS directory in terminal
```

Comment:
This is part of the "circular dependency" issue that currently exists between WEIS and the ROSCO toolbox. The ROSCO toolbox assumes that WEIS is installed (Because it needs AeroelasticSE and CCBlade), and WEIS needs the ROSCO toolbox to tune ROSCO and run aeroelastic simulations.

Reply:
Understood, thanks for spelling it out!